### PR TITLE
Kulgavykh

### PR DESCRIPTION
**Pull request by blinmakersha**
First HW.

### CODE REVIEW

File: .DS_Store
Comment (siriusdevs) [inline .DS_Store:1]: this should be ignored with .gitignore

File: .github/workflows/pylint.yml.save
Comment (siriusdevs) [inline .github/workflows/pylint.yml.save:1]: better delete this file

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 *branches.sh
-*students*
\ No newline at end of file
+*students*
+*pycache*
+*.env*
```
Comment (siriusdevs) [inline .gitignore]: *DS_Store*

File: QuizGame/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+# How to start. First steps
+    git clone https://github.com/siriusdevs/rpm-hws_7-2_2023
+
+    git checkout kulgavykh
+
+    cd quizgame
+
+# Next step is creating Docker container and filling database
+    run init_script.sh
```
Comment (siriusdevs) [inline QuizGame/README.md]: firstly
`chmod +x init_script.sh`
then
`./init_script.sh`

Comment (siriusdevs) [reply QuizGame/README.md]: Also, you init_script.sh asks password for db to finish. you can [fix this](https://stackoverflow.com/questions/6523019/postgresql-scripting-psql-execution-with-password) or just change the script logic

File: QuizGame/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+### Then, you shoul fill the .env file
+# .env
+    HOST - database host
```
Comment (siriusdevs) [inline QuizGame/README.md]: how do i know this creds? you set docker pg up yourself with your script.
maybe you could set up .env file in that script too?

**Review by siriusdevs — CHANGES_REQUESTED** [review]
fix initializing script and readme

File: QuizGame/README.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# About project "QuizGame":
+## This is a simple version of quiz written on Python that connects with database.
+
+# How to start. First steps
+    git clone https://github.com/siriusdevs/rpm-hws_7-2_2023
+
+    git checkout kulgavykh
+
+    cd quizgame
```
Comment (siriusdevs) [inline QuizGame/README.md]: also, you do not have a directory named quizgame, it is QuizGame you have

**Comment by blinmakersha** [comment]
Made some changes.

File: QuizGame/README.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+# Next step is creating Docker container and filling database
+    chmod +x init_script.sh
+
+    ./init_script.sh
+
+### Then, you shoul fill the .env file
+# .env
+    HOST - 127.0.0.1
```
Comment (siriusdevs) [inline QuizGame/README.md]: you could actually just replace " - " with "=" and it would be a legitimate env file available for copying

**Review by siriusdevs — CHANGES_REQUESTED** [review]
there is no Quiz result at the end of the game, which is sad. please kindly add some.

moreover, [banana is a language](https://github.com/KyleMiles/bananaScript) ! :smiley: 
just kidding

**Pull request merged**